src/SwedbankPay.Sdk.Tests/Json/CustomLanguageConverterTests.cs

```diff
@@ -1,47 +1,50 @@
-﻿using System;
-
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
-
-using Xunit;
-
-namespace SwedbankPay.Sdk.Tests.Json
-{
-    public class CustomLanguageConverterTests
-    {
-        private readonly string languageString = "sv-SE";
-
-
-        [Fact]
-        public void CanDeSerialize_Language()
-        {
-            //ARRANGE
-            var jsonObject = new JObject { { "language", this.languageString } };
-
-            //ACT
-            var result = JsonConvert.DeserializeObject<Language>(jsonObject.ToString(), JsonSerialization.JsonSerialization.Settings);
-
-            //ASSERT
-            Assert.Equal(this.languageString, result.ToString());
-        }
-
-
-        [Fact]
-        public void CanSerialize_Language()
-        {
-            //ARRANGE
-            var dummy = new
-            {
-                Language = new Language(this.languageString)
-            };
-
-            //ACT
-            var result = JsonConvert.SerializeObject(dummy, JsonSerialization.JsonSerialization.Settings);
-            var obj = JObject.Parse(result);
-
-            obj.TryGetValue("Language", StringComparison.InvariantCultureIgnoreCase, out var language);
-            //ASSERT
-            Assert.Equal(this.languageString, language);
-        }
-    }
+﻿using SwedbankPay.Sdk.Common;
+using System.Text.Json;
+
+using Xunit;
+
+namespace SwedbankPay.Sdk.Tests.Json
+{
+    public class CustomLanguageConverterTests
+    {
+        private readonly string languageString = "sv-SE";
+
+
+        [Fact]
+        public void CanDeSerialize_Language()
+        {
+            //ARRANGE
+            var jsonObject = $"{{ \"language\": \"{this.languageString}\" }}";
+
+            //ACT
+            var result = JsonSerializer.Deserialize<Language>(jsonObject.ToString(), JsonSerialization.JsonSerialization.Settings);

```

Comment:
Codacy found an issue: [There's no need to call 'ToString()' on a string.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)